registrations/permissions.py

```diff
@@ -88,4 +88,18 @@ def has_object_permission(
         if request.method == "DELETE":
             return obj.can_be_deleted_by(request.user)
 
+        if request.method in ("PUT", "PATCH"):
+            if obj.registration.registration_user_accesses.filter(
```

Comment:
Is there any possibility that the email of the registration creator, registration admin or superuser ends up in registration_user_accesses? 

If yes, then this could be problematic as it locks them to updating only presence_status.

The logic is maybe a bit tricky:

1) superusers and admins should always go via can_be_edited_by and skip any field restrictions
2) field restrictions apply if user is creator or email is in registration_user_accesses, but no restrictions if both?
3) fallback for other users via can_be_edited_by